package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "nahmii-sdk",
-  "version": "4.5.0",
+  "version": "4.5.1",
```

Comment:
I can see the argument for only changing the patch number despite the fact that the PR has introduced API changes.
So maybe that is the right thing to do.

Reply:
That is my rationale too.